### ISSUE TITLE
Content updates for chatbot launch from Virtual Agent Team

### ISSUE THREAD

**Issue by ianpsant** [department-of-veterans-affairs/va.gov-team#33905]
## What does your team need support for? Check all that apply.
- [ ] Launching one or more new unauthenticated (static) VA.gov pages
- [X] Making revisions to one or more existing unauthenticated (static) VA.gov page
- [X] Launching a new online form, app, or tool on VA.gov
- [ ] Making revisions to an existing online form, app, or tool on VA.gov
- [ ] Something else (please add details in the next question)

## What do you need help with?
  - [X] Content support
  - [X] IA support

## Resources
[36443](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/36443)  

*Please describe your need and provide any additional documents and/or links that will help us help you. If we need to add a react widget to a page, be sure to provide the code here.*

We will be launching the unauthenticated Virtual Agent Chatbot chatbot to a percentage of VA.gov users in February, followed by a release to the same chatbot of an authenticated user experience. We have two asks

Scenario 1: Update Contact Us page to include link to chatbot
GIVEN user is on the Contact Us page (https://www.va.gov/contact-us/)
WHEN the system selects them to be a user who should see the link to the chatbot
THEN user is presented an Info Box with a link to the chatbot page, with additional language explaining that this is just a preview/pilot/beta (exact language TBD by the content team as discussed during our meeting with them)

Scenario 2: Update URL and content within the actual chatbot page
GIVEN user is on the Contact Us page 
WHEN they click on the link to reach the chatbot page (current staging version: https://staging.va.gov/virtual-agent-study/)
THEN content on the left side of the chatbot page is updated to reflect the chatbot's preview/pilot/beta state, and the URL of this page is updated accordingly 
  
## Supporting Artifacts
*Please provide supporting artifacts as available.*

- Link to product outline:  https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/virtual-agent/Virtual_Agent_Product_Outline_Phase2.pdf
- Link to or attach user flows:
- Link to prototype or mockups: https://staging.va.gov/virtual-agent-study/
- Link to any epics/issues that may be helpful:

## Will this work be going through the Collaboration Cycle?
- [X] Yes 
- [ ] No

## When does this work need to be done?
- Estimated launch date: February 28, 2022
- Estimated staging review date: January 10, 2022
- Content and IA work needed by:  February 14, 2022

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [X] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: Virtual Agent Chatbot
-	OCTO-DE product owner:  @LucianaAlexanderMorais 
-	Product manager: @ianpsant 
-	Designer: @pilarmo 
-	FE engineer: @TanyaMaiRandstoft 
-	Product/team Slack channel: #va-virtual-agent-public

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#va-sitewide-content](https://dsva.slack.com/channels/va-sitewide-content) Slack channel and tag @RLHecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by RLHecht** [comment]
@ianpsant We were planning on looking at this in the new year. Does that time frame work?

**Comment by ianpsant** [comment]
It think after the New Year would be fine. We don't plan on going live until February. We do have a Staging Review meeting set for January 6th and possibly the only thing that would be of interest to that team would be the Info Box placement (that will contain the link to the Chatbot page) on the Contact Us page, but I think we can talk through that.

cc: @agmurphy

**Comment by mnorthuis** [comment]
Hi @ianpsant!  I apologize for not responding to this issue, I had misunderstood the plan and was thinking the chatbot would live on the existing Contact Us page.   

Since it looks like this will be living on it's own page and slowly scaled, I've start putting together some IA recommendations.  A couple questions for you:
1) If you want to control how scaled the chatbot is, the chatbot page should be set as noindex and kept out of the xml sitemap during the roll-out to keep search engines from crawling and picking it up.  Thoughts on that?  
2) What did research tell you in regards to what this chatbot is called or how it will be referenced?  That will feed into the H1 and the URL of the page.

**Comment by ianpsant** [comment]
Hi @mnorthuis! No worries misunderstanding. Things are moving quickly in advance of our launch.

1. noindex'ing this page makes sense until the Virtual Agent's existence is more widely disseminated
2. For now, we want to stick with Virtual Agent. The consensus was that we didn't want to give it a personal name (e.g. Alexa, Siri, etc). The veterans were more interested in just having something that works. So if the URL becomes something like https://www.va.gov/virtual-agent/ that would work. In case you're interested the Branding research is here: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/virtual-agent/research/moderated-interviews/virtual-agent-branding-report.md

The H1 though requires a little more conversation. @bethpotts @DanielleThierryUSDSVA If you recall, we had a meeting in December talking about how even though we're going to production, we're still treating this launch as a pilot or beta since we're still gathering data and understanding veteran data usage. Hence why the chatbot is still on a separate page. The second half of this intake ticket was to change the left side of the https://staging.va.gov/virtual-agent-study/ page where it states it's a Virtual Agent Study. We still need those changes. I'm happy to schedule a quick call this afternoon if we need to discuss again?

cc: @LucianaAlexanderMorais

**Comment by ianpsant** [comment]
Hi @RLHecht @mnorthuis @bethpotts @DanielleThierryUSDSVA! With 29 days left before our February 28th launch I wanted to check in on the various threads in this request.

1. Contact Us Alert Box and React widget to restrict being displayed to only 25% of traffic on Contact Us
  - I see the Alert Box language was mostly finalized. I just want to make sure the Public Websites team has enough lead time to schedule this to get it done before February 28th.
2. IA production URL
  - It would be good to know this so we can provide to the Analytics & Insights Team so we can make sure our Google Analytics and Domo dashboard are set up correctly. As well as our Medallia survey.
3. Chatbot page content 
  - Sounded like this will be completed in the next few days. After we see it, we may have some additional content we'll need added based on feedback, but one thing at a time.

Any updates would be appreciated. Thanks!

**Comment by mnorthuis** [comment]
@ianpsant IA information is drafted, but I won't have the URL until I have a chance to see the content page and corresponding H1.  The URL will queue off of language in the H1 and the page. I will finalize IA and the URL once the content is ready.  

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/ia-reviews/chatbot.md

**Comment by bethpottsVADEPO** [comment]
@ianpsant Thanks for checking in. 
1: I believe @RLHecht has already gotten this in the PW queue, but I'll let her give an update here on the timing of PW completing that work.
2: This is on hold until the landing page H1 is determined. The content team and Mikki will touch base on this.
3: This is in progress this sprint. I wanted to double check what we'd talked about @ianpsant: the idea is to change the way we describe the virtual agent from being a "study" to being a "beta." Is that right? Might be helpful for the content team members who have been assigned this work to meet with you to hear directly about what you're thinking for the messaging. Everything I see in staging messages the study specifically, so it seems like the whole page needs rewriting. In other words, I think it's more than just tweaking language to change "study" to "beta." Could you and @RLHecht set up time to chat this week?

**Comment by mnorthuis** [comment]
Hi @ianpsant - IA elements are finalized here: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/ia-reviews/chatbot.md

Please let me know if you have any questions!  You can close this issue out once you have the information you need.

**Comment by ianpsant** [comment]
Just wanted to provide an update on our validation efforts.

1. Contact Us Alert Box and React widget - We'll keep this ticket open until the React widget is completed and tested (which will most likely have to be in production)
2. IA prod link - I think we're good here
3. Chatbot page content - we'll be adding your written content to staging for all to review this week.

**Comment by RLHecht** [comment]
Thanks, @ianpsant Just a heads up that @bethpottsVADEPO still needs to do a final review of the content our team provided last week. What's in our content ticket isn't quite ready for you to put into staging just yet.

**Comment by ianpsant** [comment]
@RLHecht Ahh OK. Please let us know when it's ready. We have a Go/No Go meeting with stakeholders a week from tomorrow (Tues 2/22) and ideally we'd like to have that in a final state.

**Comment by RLHecht** [comment]
This work has been completed. Closing ticket.
